### ISSUE TITLE
Review ease of navigation for keyboard only users. 

### ISSUE THREAD

**Issue by Mike-Heneghan** [Mike-Heneghan/ALISS#110]
Manually test navigating the main pages to ensure that all items can be navigated and interacted with as intended.

**Comment by Mike-Heneghan** [comment]
Example of an element which can't be reached with the keyboard:

<h4 id="service_actions_toggle" class="toggle-heading active">Actions</h4>

**Comment by Mike-Heneghan** [comment]
-  [ ] On skip to search the input field should be focussed.

**Comment by Mike-Heneghan** [comment]
Should the service area map have an alt tag or maybe be aria-hidden?

https://stackoverflow.com/questions/44712753/what-is-a-suitable-wai-aria-role-attribute-for-a-map-element

Article suggests that map elements can still be useful for some screen reader users such as those with low vision. They suggest using aria-role application.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Application_Role
